docs/src/recomended-configuration.md

```diff
@@ -1,31 +1,32 @@
 ---
 layout: two-columns
-title: Recommended recommendation
+title: Recommended configuration
 ---
 
 This is great news to know that you want to integrate DocSearch in your website.
 A good search experience is key to help your users discover your content.
 
-This section, [empowered by the details regarding how we build a DocSearch
-index][1], this section will give what is the requirements in order to have a
-great experience.
+This section, [empowered by the details about how do we build a DocSearch
+index][1], gives you the best practices to optimize our crawl. It will enhance
+your user's journey.
 
 ## Recommendations
 
-- My website should have [an updated sitemap][2]. This is key in order to let us
-  know what should be updated. Do not worry, we will still crawl your website
-  and discover embedded hyperlinks to find your great content.
+- Your website should have [an updated sitemap][2]. This is key to let us know
+  what should be updated. Do not worry, we will still crawl your website and
+  discover embedded hyperlinks to find your great content.
 
-- Every pages needs to have her full context available. Using [metadata is
+- Every pages needs to have their full context available. Using [metadata is
```

Comment:
I don't think it's very clear what you mean by `metadata is meaningful`? Are you talking about specific `<meta>` tags in the HTML? If so, we should list the one we'll use